app/views/about/index.html.erb

```diff
@@ -31,6 +31,66 @@
     </div>
   </div>
   <hr>
+  <% content_for :title, "CircuitVerse - Core Members" %>
```

Comment:
This will change the page title. Likely unintended

Reply:
@gr455 Sorry, I don't understand what you're referring to?. Can you please explain me?😅

Reply:
Yeah, this line adds <title>Circuitverse....</title> when rendering which changes the title on the tab. In your video you can see on the top of the tab that the 2 titles are concatenated

Reply:
@gr455  I didn't notice that 😅,thnx for telling me, I will remove it right away, by the way can you please help me in setting up CircuitVerse instance Docker

Reply:
Yeah even I will have to look at why you're having that error. Meanwhile see if you can google anything about it